### ISSUE TITLE
Left swipe bring back previous card

### ISSUE THREAD

**Issue by kev8800** [mac-gallagher/Shuffle#152]
Can left swipe bring back previous card

**Comment by WestFlow127** [comment]
Is this possible with the current implementation?
This is a desired feature for our use case.
@kev8800

**Comment by mac-gallagher** [comment]
@WestFlow127 As in like a carousel?

**Comment by WestFlow127** [comment]
Yes. I did manage to make it work with the available methods. Thanks for responding @mac-gallagher

**Comment by mac-gallagher** [comment]
Glad it worked out! I'm curious about this - this isn't a use-case I considered. Are you able to share a code snippet?
